### PR TITLE
feat(fxa-57): reminder spec

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes https://github.com/mozilla/fxa-auth-mailer/issues/95

### CODE REVIEW

**Comment by vladikoff** [comment]
- [ ] @rfk r?
- [ ] @ryanfeeley r?

**Comment by rfk** [comment]
I still feel like we should say something about sync or about "signing into Firefox".  Most users who get this email will have had no explicit intention of "creating a Firefox Account", they will have been told to sign in to Firefox by the first-run flow.

File: features/FxA-57-verify-reminder/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Please activate your account by confirming this email address.
+
+
+## Email 2
+
+### Header: Still there?
+
+### Content:
+
+A few weeks ago you created a Firefox Account, but never verified it.
```
Comment (rfk) [inline features/FxA-57-verify-reminder/README.md]: "A few weeks" is not consistent with "the second email arrives after three days since sign up".

**Comment by vladikoff** [comment]
Updated!

File: features/FxA-57-verify-reminder/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ 
+ The second email arrives after 168 hours since sign up.
+ 
+## Basic Email Template
+
+> This is just a visual guide, please see detailed email content below
+
+![](basic-template.png)
+
+
```
Comment (ryanfeeley) [inline features/FxA-57-verify-reminder/README.md]: @mozmatej  We are going to send out reminder emails when users neglect to verify their account. We assume they were busy or never got around to it. We will send one after 48 hours and after one week. Is this copy okay?

Comment (vladikoff) [inline features/FxA-57-verify-reminder/README.md]: @mozmatej you can see the contents of emails by viewing: https://github.com/mozilla/fxa/pull/70/files

**Comment by vladikoff** [comment]
> I still feel like we should say something about sync or about "signing into Firefox". Most users who get this email will have had no explicit intention of "creating a Firefox Account", they will have been told to sign in to Firefox by the first-run flow.

Check out the changes we came up with @ryanfeeley

File: features/FxA-57-verify-reminder/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+## Email 1
+
+### Header: Hello again.
+
+### Content:
+
+A few days ago you created a Firefox Account, but never verified it.
+Once verified, you can access your tabs, bookmarks, passwords and
```
Comment (MozMatej) [inline features/FxA-57-verify-reminder/README.md]: A verified account lets you access your tabs, bookmarks, passwords and history on any device connected to it.

File: features/FxA-57-verify-reminder/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+ history on any device connected to your Firefox Account.
+ Simply activate your account by confirming this email address.
```
Comment (MozMatej) [inline features/FxA-57-verify-reminder/README.md]: Simply confirm this email address to activate your account.

File: features/FxA-57-verify-reminder/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+
+## Email 2
+
+### Header: Still there?
+
+### Content:
+
+A week ago you created a Firefox Account, but never verified it. We’re worried about you.
+Firefox is available for Windows, Mac, Linux, Android and iOS devices.
```
Comment (MozMatej) [inline features/FxA-57-verify-reminder/README.md]: I think you can remove "devices" from the end of this line.

File: features/FxA-57-verify-reminder/README.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+You can sync your Firefox passwords, bookmarks and history across all of them.
+Let us know you’re okay by activating this account.
```
Comment (MozMatej) [inline features/FxA-57-verify-reminder/README.md]: Confirm this email address to activate your account and let us know you're okay.

File: features/FxA-57-verify-reminder/README.md (hunk 5 above)
Comment (ryanfeeley) [inline features/FxA-57-verify-reminder/README.md]: Is it okay that the next sentence ends in "them"?

File: features/FxA-57-verify-reminder/README.md (hunk 3 above)
Comment (ryanfeeley) [inline features/FxA-57-verify-reminder/README.md]: "on any connected device"? I heard it in a Hulu commercial, so I'm hoping it's mainstream enough.

File: features/FxA-57-verify-reminder/README.md (hunk 6 above)
Comment (ryanfeeley) [inline features/FxA-57-verify-reminder/README.md]: :+1:

File: features/FxA-57-verify-reminder/README.md (hunk 4 above)
Comment (ryanfeeley) [inline features/FxA-57-verify-reminder/README.md]: :+1:

File: features/FxA-57-verify-reminder/README.md (hunk 5 above)
Comment (MozMatej) [inline features/FxA-57-verify-reminder/README.md]: Yup, still works.

File: features/FxA-57-verify-reminder/README.md (hunk 3 above)
Comment (MozMatej) [inline features/FxA-57-verify-reminder/README.md]: A connected device sounds to me like anything connected to the Internet. We should keep it about devices that are connected to the user's account.

**Comment by rfk** [comment]
r+, but noting that @MozMatej suggested a few re-wordings that need to be folded back into the canonical text.

We could consider sending the more generic reminders for people who *weren't* trying to sign up to sync, but it's probably to much work for not enough gain given the relative number of people in each category.

**Comment by rfk** [comment]
I also think it's worth taking an initial stab at https://github.com/mozilla/fxa-auth-server/issues/1082 and recording the proposed "success criteria" in this document for reference, but that's a separate PR.

File: features/FxA-57-verify-reminder/README.md (hunk 3 above)
Comment (vladikoff) [inline features/FxA-57-verify-reminder/README.md]: :+1:

File: features/FxA-57-verify-reminder/README.md (hunk 5 above)
Comment (vladikoff) [inline features/FxA-57-verify-reminder/README.md]: :+1:

**Pull request merged**